sources/packages/web/src/views/student/application-offering-change-request/ApplicationOfferingChangeFormView.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Financial Aid Application"
+        sub-title="View Request"
+        :routeLocation="goBackRouteParams"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestStatus ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+          "
+        >
+          <v-btn
```

Comment:
just wondering do we need to add any AEST role checks for these action buttons? @dheepak-aot @andrepestana-aot @guru-aot

Reply:
Good point. Need to ask @JasonCTang about ministry role and add it to #1981

Reply:
If I'm not mistaken, this ticket is for the view in the Student Portal, correct?  And the buttons in questions would be decline, approve and maybe the buttons in the declaration screen as well? And by roles, do we mean the KeyCloak groups?  

If my assumptions above are correct, then these buttons would only be executable by students and Ministry staff will not have access to the buttons.

Reply:
@JasonCTang , As mentioned by @dheepak-aot, we will need to add the role to the ministry story #1981 , if there is any

Reply:
@ann-aot @dheepak-aot I guess I don't really understand the question.  These buttons would not be present in the Ministry view, right? So wouldn't there be no Ministry role relevant here?